src/pe_reports/report_generator.py

```diff
@@ -55,6 +62,18 @@ def load_template():
     return prs
 
 
+def load_customers():
+    """Export PowerPoint report set to output directory."""
+    try:
+
+        if os.path.getsize(CUSTOMERS) != 0 and os.path.exists(CUSTOMERS):
```

Comment:
If you're explicitly checking that a file exists then it's the first thing that should occur.

```suggestion
    try:
        if os.path.exists(CUSTOMERS) and os.path.getsize(CUSTOMERS) != 0:
```